cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -939,6 +944,25 @@ def parse_bucket_url(url):
     return {"bucket_name": bucket_name, "object_key": object_key, "object_name": object_name}
 
 
+def generate_s3_presigned_url(s3_uri, expiration=3600):
```

Comment:
can we move this under an util package? Probably `pcluster.utils` is the best match

Reply:
Sure, I switched to the AWSApi.instance().s3 api